management/branding.go

```diff
@@ -16,10 +20,106 @@ type Branding struct {
 type BrandingColors struct {
 	// Accent color
 	Primary *string `json:"primary,omitempty"`
+
 	// Page background color
+	// Only one of PageBackground and PageBackgroundGradient should be set.
+	// If both fields are set, PageBackground takes priority.
+	PageBackground *string `json:"-"`
+
+	// Page background gradient
+	// Only one of PageBackground and PageBackgroundGradient should be set.
+	// If both fields are set, PageBackground takes priority.
```

Comment:
I would go as far as return an error during Marshal/Unmarshal if both are set. What do you think?

Reply:
During Unmarshal this is not an issue, as the JSON can only contain one of both.

For Marshal I added a check to the if-statement to see if both are `!= nil` and return an error in that case.